compiler/llvm/clangUtil.cpp

```diff
@@ -4519,6 +4523,56 @@ void makeBinaryLLVM(void) {
   }
 }
 
+static void handlePrintAsm(std::string dotOFile) {
+  if (llvmPrintIrStageNum == llvmStageNum::ASM ||
+      llvmPrintIrStageNum == llvmStageNum::EVERY) {
+
+    // find the directory containing clang
+    std::vector<std::string> split;
+    std::string llvmObjDump = "llvm-objdump";
+
+    splitStringWhitespace(CHPL_LLVM_CLANG_C, split);
+    if (split.size() > 0) {
+      std::string tmp = split[0];
+      const char* clang = "clang";
```

Comment:
`setupClang` does something that I think is much easier to follow with `sys::path::parent_path()` that I think would be nice to use here. That also makes me wonder if there's any llvm path utilities for finding a binary. [findProgramByName](https://llvm.org/doxygen/namespacellvm_1_1sys.html#a7f2dd714b9f4a359e62b257376e38d33) seems like a possibility, but consider that and `parent_path` suggestion as "would be nice future cleanup" rather than anything that must be done for this PR.